action.yml

```diff
@@ -72,13 +76,40 @@ runs:
       shell: bash
       run: npm i -g snyk-delta
 
+    - name: Get Snyk attributes
+      shell: bash
+      run: |
+        echo "snyk_org=$(jq -r .org < snyk.json)" >> "$GITHUB_ENV"
+        echo "snyk_project_name=$(jq -r .projectName < snyk.json)" >> "$GITHUB_ENV"
+        echo "snyk_target_file=$(jq -r .targetFile < snyk.json)" >> "$GITHUB_ENV"
+
+    - name: Get Snyk org ID
+      id: get_snyk_org_id
+      uses: fjogeleit/http-request-action@v1.16.1
+      with:
+        url: 'https://api.snyk.io/rest/orgs?version=${{ inputs.snyk-api-version }}&slug=${{ env.snyk_org }}'
+        method: 'GET'
+        customHeaders: '{"Content-Type": "application/vnd.api+json", "Authorization": "token ${{ inputs.snyk-token }}"}'
+        retry: 3
+
+    - name: Get baseline project ID
+      id: get_baseline_project_id
+      uses: fjogeleit/http-request-action@v1.16.1
+      with:
+        url: 'https://api.snyk.io/rest/orgs/${{ fromJson(steps.get_snyk_org_id.outputs.response).data[0].id }}/projects?version=${{ inputs.snyk-api-version }}&target_reference=${{ github.base_ref }}&names=${{ env.snyk_project_name }}%3A${{ env.snyk_target_file }}'
+        method: 'GET'
+        customHeaders: '{"Content-Type": "application/vnd.api+json", "Authorization": "token ${{ inputs.snyk-token }}"}'
```

Comment:
same here, is the `Content-Type` header here necessary? We are not sending any content.